completions/rsync

```diff
@@ -1,5 +1,36 @@
 # bash completion for rsync                                -*- shell-script -*-
 
+_rsync_vercomp() {
```

Comment:
_comp_cmd_rsync__vercomp

https://github.com/scop/bash-completion/blob/master/doc/api-and-naming.md#naming
We are still in the middle of migration, but the new functions would be better to follow the new naming.